docs/development/db-schema.md

```diff
@@ -0,0 +1,60 @@
+# Database Schema
+
+Currently Service Manager supports only PostgreSQL as a database.
+
+The database schema is built using a sequence of SQL scripts located in
+[storage/postgres/migrations](../../storage/postgres/migrations).
+For each schema version there are two scripts - _up_ and _down_.
+So it is possible also to downgrade the schema to a an older version.
+Each file name starts with some numbers - these represent the schema version.
```

Comment:
Most recent one are timestamps, as the migrations are now applied based on the timestamp ordering, instead numbers. This was introduced to allow database migrations to be added easily from SM extensions.